src/com/oltpbenchmark/api/Worker.java

```diff
@@ -63,7 +63,8 @@
 
     private boolean seenDone = false;
 
-    int totalRetries = 0;
+    int[] totalRetries = {0, 0, 0, 0, 0};
+    int[] totalFailures = {0, 0, 0, 0, 0};
```

Comment:
here and above, java int's are default zero (that's part of the standard), so prefer something like `int[] totalRetries = new int[NUM_WORKER_TYPES]`

Reply:
Done.